webapp/src/containers/MasternodesPage/saga.tsx

```diff
@@ -113,24 +113,19 @@ export function* handleRestartNode() {
       const data = yield call(getAddressInfo, masternodeOwner);
       if (data.ismine && !data.iswatchonly) {
         const updatedConf = yield call(getConfigurationDetails);
-        if (updatedConf[network]) {
+        updatedConf[network] = {
+          ...updatedConf[network],
```

Comment:
@saurabh391  isn't it still the same? If the user doesn't have updatedConf[network], it will break the spread process right? 

1. If updatedConf[network] has some values (e.g, { wallet: 'wallet.dat', walletDir: 'home/wallet' })
2. If updatedConf[network] is null or undefined, the spread operator `...` will break right?

Reply:
JJ could you suggest improvement here thanks